integration-core/src/test/java/org/symphonyoss/integration/core/bootstrap/IntegrationLoggingTest.java

```diff
@@ -98,6 +100,13 @@ public void testLogIntegrationReady() throws InterruptedException {
     assertTrue(queue.isEmpty());
   }
 
+  @Test(expected = RemoteApiException.class)
+  public void testLogIntegrationRemoteApiException() throws InterruptedException {
+    doThrow(RemoteApiException.class).when(integration).getSettings();
```

Comment:
Makes more sense change it to: doThrow(RemoteApiException.class).when(jsonUtils).serialize(any());